### ISSUE TITLE
FAQ entry about case/switch should be revamped to give more information about match...case

### ISSUE THREAD

**Issue by stevendaprano** [python/cpython#98387]
The FAQ entry [Why isn’t there a switch or case statement in Python?](https://docs.python.org/3/faq/design.html#id12) is now obsolete and should be removed or revamped.

**Comment by pochmann** [comment]
Why, does Python have a switch statement now?

**Comment by stevendaprano** [comment]
> Why, does Python have a switch statement now?

Yes, see [PEP 634](https://peps.python.org/pep-0634/), although of course it is so much more than just a mere Pascal-style case or C-style switch, but it can be used as such:

```
>>> n = 2
>>> match n:
...     case 1: print('one')
...     case 2: print('two')
...     case _: print('nothing matched')
... 
two
```

I just noticed that the FAQ entry already briefly refers to `match...case` :-)

(I got fooled by the title, and was lazy in my reading. Mea culpa.)

I suggest that we relabel the entry "Does Python have..." rather than "Why doesn't Python have...". We should keep the existing solutions (chained `if...elif` and dispatch tables), but also link to the `match` PEP, [tutorial](https://peps.python.org/pep-0636) and [the docs](https://docs.python.org/3/reference/compound_stmts.html#the-match-statement).

**Comment by pochmann** [comment]
Ok, I thought you meant something else, mostly because `match` is already mentioned there. (And because that's similar but also quite different, supporting more but also "lacking" optimization and fallthrough, if I'm not mistaken).
